homeassistant/components/honeywell/climate.py

```diff
@@ -33,49 +31,25 @@
     SUPPORT_TARGET_TEMPERATURE,
     SUPPORT_TARGET_TEMPERATURE_RANGE,
 )
+from homeassistant.config_entries import SOURCE_IMPORT
 from homeassistant.const import (
     ATTR_TEMPERATURE,
-    CONF_PASSWORD,
-    CONF_REGION,
-    CONF_USERNAME,
+    PERCENTAGE,
     TEMP_CELSIUS,
     TEMP_FAHRENHEIT,
 )
-import homeassistant.helpers.config_validation as cv
 
-_LOGGER = logging.getLogger(__name__)
+from .const import (
+    _LOGGER,
+    CONF_COOL_AWAY_TEMPERATURE,
+    CONF_HEAT_AWAY_TEMPERATURE,
+    DOMAIN,
+)
 
 ATTR_FAN_ACTION = "fan_action"
 
-CONF_COOL_AWAY_TEMPERATURE = "away_cool_temperature"
-CONF_HEAT_AWAY_TEMPERATURE = "away_heat_temperature"
-CONF_DEV_ID = "thermostat"
-CONF_LOC_ID = "location"
-
-DEFAULT_COOL_AWAY_TEMPERATURE = 88
-DEFAULT_HEAT_AWAY_TEMPERATURE = 61
-
 ATTR_PERMANENT_HOLD = "permanent_hold"
 
-PLATFORM_SCHEMA = vol.All(
```

Comment:
Leave the existing schema and rename `setup_platform` to `async def async_setup_platform`, then use it to start an import flow so users don't have to resetup the integration.

Search the codebase for SOURCE_IMPORT for an example

Reply:
Is there any reason I need to leave the existing schema in here? `async_setup_platform` is already defined below on line 103 and I've testing the import flow, seems to be working as expected. Unless I'm missing something?

Reply:
We would not have validation if we drop the schema. It may or may not save some work in the import step.

Reply:
Added it back.